tGUI.m

```diff
@@ -1,33 +1,33 @@
 %
-% tGUI(EXPERIMENT_NAME_STRING)
+% tGUI;
 %
-% Opens a Thermoesthesia GUI (tGUI) with figure title EXPERIMENT_NAME_STRING.
+% Waits for the arduino to read thermoneutral before it opens a 
```

Comment:
Consider changing the text for ln 4 to the following: "Reads input from two temperature sensors connected to an Arduino Uno. When the sensors values are approximately equal (+/- 2 degrees Celsius), opens a ...